substrate/frame/treasury/src/lib.rs

```diff
@@ -240,6 +242,9 @@ pub mod pallet {
 		///
 		/// NOTE: This parameter is also used within the Bounties Pallet extension if enabled.
 		#[pallet::constant]
+		#[deprecated(
+			note = "Gov v1 type used for `spend_local`, configure pallet to use `PayFromAccount` for `Paymaster` type and use `spend` instead"
```

Comment:
```suggestion
			note = "The `MaxApprovals` config type will be removed by May 2025. It associated with the deprecated `spend_local` call."
```
if no span, a developer wont see which type deprecated, so better to mention the name. also your line probably do not fit into `100` max weight